src/kakarot/state.cairo

```diff
@@ -130,7 +130,11 @@ namespace State {
 
         // Transfers
         let (native_token_address_) = native_token_address.read();
-        Internals._transfer_eth(native_token_address_, self.transfers_len, self.transfers);
+        let accounts = self.accounts;
+        Internals._transfer_eth{accounts=accounts}(
+            native_token_address_, self.transfers_len, self.transfers
+        );
+        dict_squash(self.accounts_start, accounts);
```

Comment:
we weren't squashing the account dict before?

Reply:
yes, but we read again, so we need to squash again

Reply:
is it "expensive" to squash twice?

Reply:
I don't know the cost of squashing, but it's required in any case, and not optimizable so much other than optimizing the number of read. In any case, for this PR I've removed all this